docs/OV_Runtime_UG/supported_plugins/CPU.md

```diff
@@ -1,139 +1,147 @@
+The most flexible configuration is the fully undefined shape, when we do not apply any constraints to the shape dimensions. 
 
-![conv_depth_01]
+@snippet snippets/cpu/dynamic_shape.cpp undefined_shape
 
-### Fusing Convolution and Sum Layers
+In such configuration we will have high memory consumption since we can not estimate the total memory amount that can be allocated in advance and effectively reused.
+To reduce memory consumption through memory reuse, and as a result achieve better cache locality, which in its turn leads to better inference performance, it is better to use dynamic shapes with defined upper bounds.
 
-A combination of convolution, simple, and Eltwise layers with the sum operation results in a single layer called *Convolution*:  
+@snippet snippets/cpu/dynamic_shape.cpp defined_upper_bound
 
-![conv_sum_relu_01]
+> **NOTE**: Some runtime optimizations works better if the model shapes are known in advance. This means that for the best performance, it is better to use static shapes, of course if it is applicable to the specific problem.
 
-### Fusing a Group of Convolutions
+See [dynamic shapes guide](../ov_dynamic_shapes.md) for more details.
 
-If a topology contains the following pipeline, a CPU plugin merges split, convolution, and concatenation layers into a single convolution layer with the group parameter:   
+### Preprocessing acceleration
+CPU plugin supports a full set of the preprocessing operations, providing high performance implementations for them.
 
-![group_convolutions_01]
+See [preprocessing API guide](../preprocessing_overview.md) for more details.
 
-> **NOTE**: Parameters of the convolution layers must coincide.
+The CPU plugin precision conversion operation implementation supports the following element types:
+- bf16
+- f16
+- f32
+- f64
+- i8
+- i16
+- i32
+- i64
+- u8
+- u16
+- u32
+- u64
+- boolean
 
+### Models caching
+CPU plugin supports Import/Export network capability. If the model caching is enabled via common OpenVINO `ov::cache_dir` property, the plugin will automatically create a cached blob inside the specified directory during model compilation.
+This cached blob contains some intermediate representation of the network that it has after common runtime optimizations and low precision transformations.
+The next time the model is compiled, the cached representation will be loaded to the plugin instead of the initial IR, so the aforementioned transformation steps will be skipped.
+These transformations take a significant amount of time during model compilation, so caching this representation reduces time spent for subsequent compilations of the model,
+thereby reducing first inference latency (FIL).
 
-### Removing a Power Layer
+See [model caching overview](@ref openvino_docs_IE_DG_Model_caching_overview) for more details.
 
-CPU plugin removes a Power layer from a topology if it has the following parameters:
-  - <b>power</b> = 1
-  - <b>scale</b> = 1
-  - <b>offset</b> = 0
+### Extensibility
```

Comment:
I guess we also should add a note that customization for operation with internal dynamism is not supported